.github/workflows/test.yaml

```diff
@@ -0,0 +1,55 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches:
+      - devel
+      - stable
```

Comment:
```suggestion
      - master
```

Reply:
Oh sorry, I copied from another project, and changed it in the wrong place! :sweat_smile: